pkg/security/resolvers/usergroup/passwd.sample

```diff
@@ -0,0 +1,3 @@
+   # messagebus:x:103:106::/nonexistent:/usr/sbin/nologin
```

Comment:
I think this could be in directory called `testdata` so it's not built into the package. 

From `go help packages`
```
Directory and file names that begin with "." or "_" are ignored
by the go tool, as are directories named "testdata".
```